components/ipfs/ipfs-cluster/BUILD.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
+  - name: docker
+    type: docker
+    config:
+      dockerfile: leeway.Dockerfile
+      buildArgs:
+        VERSION: 1.0.4
```

Comment:
I love this idea :+1:

Reply:
But I haven't found a way to pass parameters across leeway configuration, so we only need to bump the version in one place.
- https://github.com/gitpod-io/gitpod/blob/408619e8fe3253df2654b7af9b5b1a2d786f6c21/components/ipfs/ipfs-cluster/BUILD.yaml#L7
- https://github.com/gitpod-io/gitpod/blob/408619e8fe3253df2654b7af9b5b1a2d786f6c21/components/ipfs/ipfs-cluster/BUILD.yaml#L9